atmo/static/js/base.js

```diff
@@ -77,10 +77,76 @@ $(function() {
     });
   }
 
-  AtmoCallbacks.add(atmoTooltips);
+  var atmoWhatsNew = function() {
+    // Fill modal with content from link href
+    $('#whatsnew-modal').on('show.bs.modal', function(e) {
+        var link = $(e.relatedTarget);
+        $(this).find('.modal-body').load(link.attr('href'));
+    });
+    var checker = $('#whatsnew-check'),
+        checker_url = checker.attr('data-url');
+    $.get(checker_url).done(function(data) {
+      if (data !== 'ok') {
+        checker.removeClass('hidden');
+      }
+      checker.closest('li').removeClass('hidden');
+    });
+  };
+
+  var atmoTime = function() {
+    var time = $('#time'),
+        utc_now = function() {
+          return moment().utcOffset(0).format('YYYY-MM-DD HH:mm:ss');
+        };
+    var updateTime = function() {
+      time.attr('data-content', utc_now());
+      window.setTimeout(updateTime, 1000);
+    }
+    updateTime();
+  };
+
+  var atmoModifiedDate = function() {
+    var timeout_id,
+        modified_date = $('body').attr('data-modified-date'),
+        parsed_modified_date = moment(modified_date);
+    // don't continue if there is no modified date in the body attributes or the value is invalid
+    if (jQuery.type(modified_date) === "undefined" || !parsed_modified_date.isValid()) {
+      return;
+    };
+    var updateModifiedDate = function() {
+      $.ajax({
+        url: window.location,
+        type: 'head',
+        success: function(res, code, xhr) {
+          var returned_modified_date = moment(xhr.getResponseHeader("X-ATMO-Modified-Date"));
+          if (!returned_modified_date.isValid()) {
+            console.log(returned_modified_date);
```

Comment:
If we leave this in maybe add some more descriptive text around it so we know what it is in the console.